### PR TITLE
Fun value (fixes #7)

### LINKED ISSUE #7: Make "Fun" variable editable

### PR DESCRIPTION
**Pull request by jD91mZM2**

### CODE REVIEW

**Comment by jD91mZM2** [comment]
bump?  
(He accepted all my other pulls)

File: main.js [hunk 1]
```diff
@@ -645,6 +648,10 @@ function updateIniFromForm(ini) {
             ini.reset.s_key = "0";
         }
     }
+    var fun = parseInt(document.getElementById("ini-fun").value);
+    if(fun){
```
Comment (crumblingstatue) [inline main.js]: Small nitpick, but can you keep the formatting consitent? ( `if (fun) {` )

**Review by crumblingstatue — CHANGES_REQUESTED** [review]

File: main.js (hunk 1 above)
Comment (jD91mZM2) [reply main.js]: Sorry, yes! Will do tomorrow. If you don't feel like changing it. I left the "allow edits from maintainers" checkbox on

**Comment by jD91mZM2** [comment]
Fixed!

**Review by crumblingstatue — APPROVED** [review]

**Pull request merged**